tests/ci/android/README.md

```diff
@@ -0,0 +1,19 @@
+# Android CI for AWS-LC
+AWS-LC wants to ensure that our tests and build work correctly on Android. Insipred from the CRT Team's [`AWSCRTAndroidTestRunner`](https://github.com/awslabs/aws-c-common/tree/main/AWSCRTAndroidTestRunner), this mini Android test harness is intended to be used within our Android CI with AWS Device Farm. The tests will include `crypto_test`, `urandom_test`, `decrepit_test` and `ssl_test`, referenced from regular test dimensions. This app can be tested with the docker image at `tests/ci/docker_images/linux-x86/ubuntu-20.04_android`.
+
+## Setup
+1. `docker build -t ubuntu-20.04:android ../docker_images/linux-x86/ubuntu-20.04_android/`
+2. `cd AWSLCAndroidTestRunner`
```

Comment:
Nit: add below to let user know these commands should be executed inside the image with the code?
```
docker run -it -v `pwd`:`pwd` -w `pwd` {img_tag}
```